Packs/Core/Playbooks/playbook-Scheduled_task_created_with_HTTP_or_FTP_reference.yml

```diff
@@ -0,0 +1,1014 @@
+                  simple: "3"
+          accessor: Indicator
+          transformers:
+          - operator: uniq
+      pre-post:
+        simple: pre-rulebase
+      type:
+        simple: URL List
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 30,
+          "y": 2180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: e5e0d51f-b834-47d0-81f2-326aaab123dc
+    type: regular
+    task:
+      id: e5e0d51f-b834-47d0-81f2-326aaab123dc
+      version: -1
+      name: Disable the malicious scheduled task manually
+      description: "Dear Analyst,\nPlease note that during the remediation process, the playbook failed to disable the scheduled task ${ExtractedTaskName}\n\nPlease take manual action to disable the scheduled task. "
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "23"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": 1340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: c5219f31-047d-4cee-888e-f7c63909a296
+    type: title
+    task:
+      id: c5219f31-047d-4cee-888e-f7c63909a296
+      version: -1
+      name: Block Malicious Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "17"
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1520
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: e16a5d0b-f119-4691-811e-28c3d0221004
+    type: condition
+    task:
+      id: e16a5d0b-f119-4691-811e-28c3d0221004
+      version: -1
+      name: Analyst approval for Block URL
+      description: Analyst approval required for URL blocking.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "No":
+      - "13"
+      "Yes":
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": 2010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: 'Block the following malicious URL: ${BadUrl}   using PAN-OS?'
```

Comment:
We also need to check how does it look 'Should Block the following malicious URL: ${BadUrl}   using PAN-OS?' with the ingested URLs

Reply:
![Uploading image.png…]()